flaml/ml.py

```diff
@@ -175,16 +175,31 @@ def metric_loss_score(
                         metric_name
                     ].mid.fmeasure
                 elif metric_name == "seqeval":
-                    y_true = [
-                        [x for x in each_y_true if x != -100] for each_y_true in y_true
+                    st = time.time()
+
+                    zip_pred_true = [
+                        [(p, lb) for (p, lb) in zip(prediction, label) if lb != -100]
+                        for (prediction, label) in zip(y_predict, y_true)
                     ]
                     y_pred = [
-                        y_predict[each_idx][: len(y_true[each_idx])]
-                        for each_idx in range(len(y_predict))
+                        [labels[p] for (p, l) in each_list]
+                        for each_list in zip_pred_true
+                    ]
+                    y_true = [
+                        [labels[l] for (p, l) in each_list]
+                        for each_list in zip_pred_true
                     ]
+
+                    ed = time.time()
+
                     score = metric.compute(predictions=y_pred, references=y_true)[
-                        "overall_accuracy"
+                        "overall_f1"
                     ]
+
+                    ed2 = time.time()
+
+                    print("time for metric.compute:", ed2 - ed)
+                    print("time for preparing y_pred and y_true:", ed - st)

```

Comment:
remove/comment the code for measuring time.
add comment about why str labels are necessary.